sentry_sdk/integrations/fastapi.py

```diff
@@ -62,6 +65,21 @@ def patch_get_request_handler():
 
     def _sentry_get_request_handler(*args, **kwargs):
         # type: (*Any, **Any) -> Any
+        dependant = kwargs.get("dependant")
+        if dependant and not asyncio.iscoroutinefunction(dependant.call):
```

Comment:
Yes I think so.
We [vendored this](https://github.com/getsentry/sentry-python/blob/dfb04f594f7790b54f7fbdab93f407f70dd2d204/sentry_sdk/integrations/starlette.py#L330-L339) from starlette for doing these coroutine checks.

@antonpirker knows this stuff better since he worked on it.